--- a/src/fill_ringbuffer.c	2018-06-14 08:05:12.255671928 +0000
+++ b/src/fill_ringbuffer.c	2018-06-14 08:07:12.701552932 +0000
@@ -611,10 +611,10 @@
     packet = &packet_buffer[packet_idx];
 
     // check marker byte
-    if (packet->marker_byte != expected_marker_byte) {
-      LOG("ERROR: wrong marker byte: %x instead of %x\n", packet->marker_byte, expected_marker_byte);
-      clean_exit(0);
-    }
+    //if (packet->marker_byte != expected_marker_byte) {
+    //  LOG("ERROR: wrong marker byte: %x instead of %x\n", packet->marker_byte, expected_marker_byte);
+    //  clean_exit(0);
+    //}
 
     // check version
     if (packet->format_version != 1) {
@@ -630,8 +630,13 @@
 
     // check tab index 
     if (packet->tab_index >= ntabs) {
-      LOG("ERROR: unexpected tab index %d\n", packet->tab_index);
-      clean_exit(0);
+      // Ignore this packet if mode is I+IAB (current cluster setup)
+      if (science_mode == 2) {
+        continue;
+      } else {
+          LOG("ERROR: unexpected tab index %d\n", packet->tab_index);
+          clean_exit(0);
+      }
     }
 
     // check channel
